lib/locale/pl/strings.php

```diff
@@ -24,6 +24,9 @@
  *  $Id$
  */
 
+$_LANG['Verified'] = 'Weryfikowany';
+$_LANG['Ticket has been sent ticket to verificator'] = 'Zgłoszenie zostało wysłane do weryfikatora';
+$_LANG['Send ticket to verificator'] = 'Przekaż zgłoszenie do weryfikatora';
```

Comment:
Send czy transfer?

Reply:
Oba tłumaczenia pasują. Send bo weryfikator dostaje powiadomienie. Transfer bo weryfikator musi zamknąć zgłoszenie.

Reply:
Ale skoro przekaż po polsku dajesz to może lepiej transfer ticket to verifier.
Swoją drogą raz jest verifier innym razem verificator. Proszę o ujednolicenie.